src/reactviews/common/locConstants.ts

```diff
@@ -313,6 +313,8 @@ export class LocConstants {
             clear: l10n.t("Clear"),
             search: l10n.t("Search..."),
             close: l10n.t("Close"),
+            maximize: l10n.t("Maximize"),
```

Comment:
Maybe we should be more clear with these terms since "maximize" can also be confused for the "open in new tab" feature...  What do you think about "Show only this result set" and "Show all result sets"?

Reply:
> Maybe we should be more clear with these terms since "maximize" can also be confused for the "open in new tab" feature... What do you think about "Show only this result set" and "Show all result sets"?
> 
> CC: @jdesai-24

I chatted with Janki about this and we both think that sticking with "Maximize" and "Restore" is preferable for now and that we can monitor any future feedback on this